contracts/truefi2/strategies/CurveYearnStrategy.sol

```diff
@@ -143,7 +143,7 @@ contract CurveYearnStrategy is UpgradeableClaimable, ITrueStrategy {
      */
     function withdraw(uint256 minAmount) external override onlyPool {
         // get rough estimate of how much yCRV we should sell
-        uint256 roughCurveTokenAmount = calcTokenAmount(minAmount);
+        uint256 roughCurveTokenAmount = calcTokenAmount(minAmount, false);
```

Comment:
Ditto above, but checking for a call to `curve().calc_token_amount(amount, false)`, and added under the withdraw section: https://github.com/trusttoken/smart-contracts/blob/65b8699c4b44e5e18b4ff733ca4e0ccc52070ff6/test/truefi2/strategies/CurveYearnStrategy.test.ts#L136